packages/clerk-js/src/ui/components/OrganizationSwitcher/OtherOrganizationActions.tsx

```diff
@@ -1,5 +1,4 @@
-import type { OrganizationResource } from '@clerk/types';
```

Comment:
🔧 Consider splitting this big component into smaller, more dedicated components.

Reply:
`<OrganizationActionList/>` has been split into 

```tsx
<UserMembershipList />
<UserInvitationList />
<CreateOrganizationButton />
```